From 4226da330317d348093ca1ea4efa06956ad3416b Mon Sep 17 00:00:00 2001
From: Joseph Guo <qijian.guo@nxp.com>
Date: Wed, 11 Jun 2025 12:00:39 +0900
Subject: [PATCH 08/56] LF-15598: arm64: configs: imx_v8_defconfig: add
 drm_waveshare_bridge

Add CONFIG_DRM_WAVESHARE_BRIDGE to support waveshare dsi2dpi bridge by defualt.

Signed-off-by: Joseph Guo <qijian.guo@nxp.com>
Reviewed-by: Dong Aisheng <aisheng.dong@nxp.com>
---
 arch/arm64/configs/imx_v8_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index 48d55a81f4ae..7e409007fb9d 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -706,6 +706,7 @@ CONFIG_DRM_PL111=m
 CONFIG_DRM_XEN_FRONTEND=y
 CONFIG_DRM_LIMA=m
 CONFIG_DRM_PANFROST=m
+CONFIG_DRM_WAVESHARE_BRIDGE=y
 CONFIG_FB=y
 CONFIG_FB_ARMCLCD=y
 CONFIG_FB_EFI=y
-- 
2.34.1

